scripts/publish.sh

```diff
@@ -39,21 +39,21 @@ elif [ -n "$AZURE_DEVOPS_PERSONAL_ACCESS_TOKEN" ]; then
     fi
 fi
 
-if [ "$PRODUCTION" = "1" ] && [ "$NPM_TAG" = "dev" ]; then
+if [ "$PRODUCTION" = "1" ] && [ "$RELEASE_CHANNEL" = "dev" ]; then
     echo "Sync with ${GITHUB_REF} and push to it"
     git pull github "${GITHUB_REF}" --ff-only
-    git tag -a "dev/$NEXT_EXTENSION_VERSION" -m "dev/$NEXT_EXTENSION_VERSION" -m "Prisma version: $PRISMA_VERSION"
+    git tag -a "dev/$NEXT_EXTENSION_VERSION" -m "dev/$NEXT_EXTENSION_VERSION" -m "Prisma version: $NPM_VERSION"
     git push github HEAD:"${GITHUB_REF}" --follow-tags
-elif [ "$PRODUCTION" = "1" ] && [ "$NPM_TAG" = "latest" ]; then
+elif [ "$PRODUCTION" = "1" ] && [ "$RELEASE_CHANNEL" = "latest" ]; then
     echo "Sync with ${GITHUB_REF} and push to it"
     git pull github "${GITHUB_REF}" --ff-only
 
     # In the stable channel, we just need to commit the prisma_version_stable file
     # to be able to track the Prisma version against which the current stable channel extension was published
     git reset origin/master
     git add ./scripts/prisma_version_stable
-    git commit -m "bump prisma_version to $PRISMA_VERSION"
-    git tag -a "$NEXT_EXTENSION_VERSION" -m "$NEXT_EXTENSION_VERSION" -m "Prisma version: $PRISMA_VERSION"
+    git commit -m "bump npm_version to $NPM_VERSION"
```

Comment:
This first one is a file name

Reply:
yes, but also it is a parameter given by `check-update.sh`, which is called `NPM_VERSION` there.